ukaea/fair-mast
# Add some examples of using data from the archive

I am adding a notebook to the documentation that shows how to use different data we have in the archive.

Note that this PR cannot be merged until #47 is merged.

To test:
 - Run the site locally and go to http://localhost:8081/data_examples.html
 - Run the notebook and check that all the outputs are sensible.
